pom.xml

```diff
@@ -36,11 +36,9 @@
 
   <modules>
     <module>documentation</module>
-    <module>configurator</module>
```

Comment:
configurator is still required to build `devonfw-ide-scripts`.

Reply:
Added again